files/ja/web/html/global_attributes/lang/index.md

```diff
@@ -121,7 +121,7 @@ WCAG 成功基準 3.1.2 は、異なる言語の**部分**があるページに
     </li>
     <li>
       <a
-        href="/en-US/docs/Web/HTML/Global_attributes/lang$locales"
+        href="/ja/docs/Web/HTML/Global_attributes/lang$locales"
```

Comment:
I think this is a situation where you are adding a translation to an English page, so I think it is more appropriate to leave it as `en-US`.